nodes/ext_container/ext_container.go

```diff
@@ -34,6 +34,7 @@ func (s *extcont) Init(cfg *types.NodeConfig, opts ...nodes.NodeOption) error {
 	for _, o := range opts {
 		o(s)
 	}
+	s.Cfg.SkipUniquenessCheck = true
```

Comment:
I'd add a comment here explaining why this is set to true

Reply:
done. please check